PyPoE/cli/exporter/wiki/parsers/item.py

```diff
@@ -3798,6 +3800,23 @@ def _export(self, parsed_args, items):
 
         return r
 
+    def _print_item_rowid(self, parsed_args, base_item_type):
+        #Don't print anything if not running in the rowid mode.
+        if (not 'start' in vars(parsed_args).keys()) or (not 'end' in vars(parsed_args).keys()):
```

Comment:
This may be cleaner to write as:

```py
if not set(["start", "end"]) & set(vars(parsed_args).keys()):
```

For instance:

```py
t = dict(one=1, two=2, three=3)

if not set(["start", "end"]) & set(t.keys()):
    print("no start or end")
```
results in `set()` which of course satisfies falseness.

Whereas if you changed `t` to have `start` or `end` in there, it would result in `{'start'}` for instance.

Reply:
Based on your suggestion, I changed it to test if `set(["start","end"])` is a subset of `vars(parsed_args).keys()`.
This requires `parsed_args` to have a start and end value, instead of just one of those (which is what the intersection would have enforced)